connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java

```diff
@@ -109,6 +109,8 @@
 public class DistributedHerder extends AbstractHerder implements Runnable {
     private static final Logger log = LoggerFactory.getLogger(DistributedHerder.class);
 
+    private static final long FORWARD_REQUEST_SHUTDOWN_TIMEOUT_MS = 10_000L;
+    private static final long START_AND_STOP_SHUTDOWN_TIMEOUT_MS = 1000L;
```

Comment:
I'd prefer that we use `TimeUnit.SECONDS.toMillis(10)` and `TimeUnit.SECONDS.toMillis(1)` here. It avoids having to do any math when looking at the code.

Reply:
Thanks! It is better than pure number.